### PR TITLE
fix: Add feedbackQA dataset

### PR DESCRIPTION
**Pull request by cslizc**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples=None,
+        avg_character_length=None,
```
Comment (xhluca) [inline mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:32]: Would be good to add n_samples and avg_character_length. they can be computed using this script: https://github.com/McGill-NLP/statcan-dialogue-dataset/blob/ef89f82c66fab1f4ee810dd8c4e0502e6f0e08c5/retrieval_task/analyze.py

**Review by xhluca — CHANGES_REQUESTED** [review]
A few things to improve regarding metadata

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    booktitle = "Findings of the Association for Computational Linguistics: ACL 2022",
+    month = may,
+    year = "2022",
+    address = "Dublin, Ireland",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.findings-acl.75",
+    doi = "10.18653/v1/2022.findings-acl.75",
+    pages = "926--937"
+}
+"""
```
Comment (xhluca) [inline mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:50]:
```suggestion
""",
```

Need a comma here

**Review by xhluca — CHANGES_REQUESTED** [review]
Fixed comma issue

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        reference="https://arxiv.org/abs/2204.03025",
+        dataset={
+            "path": "lt2c/fqa",
+            "revision": "1ee1cd0",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="precision_at_1",
```
[the remark this answers is not in the document]
Comment (cslizc) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:20]: yes we used exact match score.

**Comment by xhluca** [comment]
@vaibhavad seems good, perhaps we can renable tests?

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:53]: re-opening this as it needs to be added

Comment (xhluca) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:53]: @cslizc fyi

**Comment by KennethEnevoldsen** [comment]
I have enabled tests - which fails due to the metadata not being filled

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py (hunk 1 above)
Comment (cslizc) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py]: added. thanks!

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
```
[the remark this answers is not in the document]
Comment (xhluca) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:27]: @cslizc can you accept this?

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        annotations_creators=None,
```
[the remark this answers is not in the document]
Comment (xhluca) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:28]: Please add this too

**Review by xhluca — CHANGES_REQUESTED** [review]
Once you accept+commit the two suggestions i think the tests should pass. might need to run linting again.

File: mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (xhluca) [reply mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py:27]: image

**Review by xhluca — APPROVED** [review]
When tests pass and points have been added (see [this file](https://github.com/xhluca/mteb/blob/82b29224959db8840be1b2254416f5c890ec4e5e/docs/mmteb/points/750.jsonl)), I think this should be good to merge

**Comment by KennethEnevoldsen** [comment]
A pull from main should resolve issues related to missing dataset references

**Review by isaac-chung — APPROVED** [review]

**Comment by isaac-chung** [comment]
@cslizc @xhluca thanks for your contributions!

**Comment by xhluca** [comment]
Thanks for the merge Isaac!

**Pull request merged**